addon/components/o-s-s/nav-tab.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="fx-row fx-1 tab-container">
+  {{#each @tabArray as |tab|}}
+    <div class="fx-col tab {{if tab.selected "tab--selected"}} {{if tab.disabled "tab--disabled"}}"
+         {{on "click" this.onSelectTab}} type="button" tabindex="0">
+      <div class="fx-row tab-content fx-gap-px-9">
+        {{#if tab.icon}}
+          <i class={{tab.icon}} />
+        {{/if}}
+        {{#if tab.label}}
+          <span>{{tab.label}}</span>
+        {{/if}}
+        {{#if tab.infoCircle}}
+          <i class="far fa-info-circle infoCircle" />
```

Comment:
`far fa-info-circle` is already a precise enough class. we can use it instead of adding an additiona `infoCircle`